benchmarks/allocation/alloc.cc

```diff
@@ -11,8 +11,12 @@ using Debug = ConditionalDebug<DEBUG_ALLOCBENCH, "Allocator benchmark">;
  */
 void __cheri_compartment("allocbench") run()
 {
+	// Make sure sail doesn't print annoying log messages in the middle of the
```

Comment:
Is it worth noting that this could be removed once Sail makes s release with this fix: https://github.com/rems-project/sail/commit/362254783e2c0a12b5393169d883ebe5621baa6f